data_structures/hashing/quadratic_probing.py

```diff
@@ -1,3 +1,26 @@
+
```

Comment:
```suggestion
```

Reply:
Move #!/usr/bin/env python3 to the first line